### ISSUE TITLE
On the question of why Sinkhorn's algorithm only needs to pass logits in the actual code, but not the prior vector ρ

### ISSUE THREAD

**Issue by zxxwmxxn** [nayeemrizve/TRSSL#9]
Hello！
         Thank you for your great work.I have a question that in the pseudo-code of the paper Sinkhorn's algorithm needs to pass two parameters logits and a priori distribution vector ρ, but in the real code implementation I found that only one parameter logits is passed. so I would like to ask you why it is possible to do so?
        Your code is as follows：
`    def forward(self, logits):
        # get assignments
        q = logits / self.epsilon
        M = torch.max(q)
        q -= M
        q = torch.exp(q).t()
        return self.iterate(q)`
